packages/gestalt/src/Button.css

```diff
@@ -53,6 +53,14 @@
   padding: var(--sema-space-150) var(--sema-space-300);
 }
 
+a.smVr:focus {
+  padding: var(--sema-space-100) calc(var(--sema-space-200) + 2px);
+}
+
+a.lgVr:focus {
+  padding: var(--sema-space-150) calc(var(--sema-space-300) + 2px);
```

Comment:
what are these random  2px?

Reply:
Those are offsets for the focus inner border, to fix the shifting issue, I should have used tokens here instead though, will fix

Reply:
DONE